features/fundraiser_can_create_a_campaign.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Create campaign
+    As an Fundraiser
+    In order to raise funds
+    I would like to be able to create a campaign 
+
+    Background: 
+      Given I visit the site
+      And I click "Create a Campaign" 
```

Comment:
Be consistent with use of single quotes, look through the entire feature file

Reply:
okay we have cleared this up and they are all now the same